resources/views/docs/desktop/1/publishing/updating.md

```diff
@@ -88,3 +94,79 @@ If you don't want your application to check for updates, you can disable the upd
 ```dotenv
 NATIVEPHP_UPDATER_ENABLED=false
 ```
+
+## Manually checking for updates
+
+You can manually check for updates by calling the `checkForUpdates` method on the `AutoUpdater` facade:
+
+```php
+use Native\Laravel\Facades\AutoUpdater;
+
+AutoUpdater::checkForUpdates();
+```
+
+**Note:** If an update is available, it will be downloaded automatically. Calling `AutoUpdater::checkForUpdates() twice
```

Comment:
Left a comment about this in the electron PR

Idea: could it be more flexible by splitting it up in checkForUpdates & download, where the first only checks, but does not download it automatically?

This way the dev has more control over behaviour. Just a thought, it's non-blocking